allennlp/common/cached_transformers.py

```diff
@@ -0,0 +1,101 @@
+import logging
+from typing import NamedTuple, Optional, Dict, Tuple
+import transformers
+from transformers import AutoModel
+
+
+logger = logging.getLogger(__name__)
+
+
+class TransformerSpec(NamedTuple):
+    model_name: str
+    override_weights_file: Optional[str] = None
+    override_weights_strip_prefix: Optional[str] = None
+
+
+_model_cache: Dict[TransformerSpec, transformers.PreTrainedModel] = {}
+
+
+def get(
+    model_name: str,
+    make_copy: bool,
+    override_weights_file: Optional[str] = None,
+    override_weights_strip_prefix: Optional[str] = None,
+) -> transformers.PreTrainedModel:
+    """
+    Returns a transformer model from the cache.
+
+    Note: If you want to re-train the weights you get from here, you should
```

Comment:
```suggestion
    Note: If you want to re-train the weights you get from here, you should.
```

Reply:
Uh, I think that's a left-over comment. It just needs to be deleted.